driver-core/src/test/resources/unified-test-format/crud/bulkWrite-comment.json

```diff
@@ -122,7 +122,6 @@
       "expectEvents": [
         {
           "client": "client0",
-          "note": "Events split due to : JAVA-4537 ",
```

Comment:
Since we had previously decided to change this test, let's not reverse this decision until we have a better alternative.  I'd prefer to revert the change to this file and leave it as is.